backend/src/app/rpc/mutations/verify_token.clj

```diff
@@ -100,11 +101,21 @@
           :opt-un [:internal.tokens.team-invitation/member-id]))
 
 (defn- accept-invitation
-  [{:keys [conn] :as cfg} {:keys [member-id team-id role] :as claims}]
-  (let [params (merge {:team-id team-id
+  [{:keys [conn] :as cfg} {:keys [member-id team-id role member-email] :as claims}]
+  (let [
+        member (profile/retrieve-profile conn member-id)
+        invitation (db/exec-one!
```

Comment:
Prefer use here the `db/get-by-param`
Approx example:

```clojure
(db/get-by-param conn :team-invitation 
                 {:team-id ... :email-to ...}
                 {:check-not-found false})
;; or

(-> (db/query conn :team-invitation {:team-id ... :email-to ...}) first)

```

Reply:
fixed